integration-tests/data/npm_sfj_2_0_2.json

```diff
@@ -0,0 +1,16 @@
+{
+  "dependencies": {
```

Comment:
`npm_sfj_2_0_2.json` What does it mean..? Are you following any naming convention.?

Reply:
Yes, I saw test data files are names as <ecosystem>_<package name>_<package version>. So followed the same.

Reply:
so in that case its `svg`right..? Package name..? not `sfj`

Reply:
svg filter I took as sf and 'j' to represent javascript.